lib/registry.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var Report = require('metrics').Report,
+  util = require('util');
+
+var Registry = module.exports = function (trackedMetrics) {
```

Comment:
Confusing construction.  We generally want to avoid chain assignment.
Avoid util.inherits(),  switching to class-style syntax preferred for this type of code:
```javascript
class Registry extends Report {
  constructor(trackedMetrics) { 
    super(trackedMetrics);
    ... 
  }
  addTaggedMetric(metricName, metric, tags) { ... }
  ...
}
module.exports = Registry
```